.github/ISSUE_TEMPLATE/issue_review_template.md

```diff
@@ -10,18 +10,29 @@ assignees: ''
 **Release PR or issue to review**
 <!-- Put here the link(s) to the issue(s) or PR(s) that need to be reviewed -->
 
-**Reviewer actions**
+**Review actions**
 
-To review, assign this issue to yourself and follow the below list.
-For any item, review the file(s) in the issue/PR listed above. 
-Put comments in the above issue or PR if they concern non-changed text.
-When done, tick the box in this issue. 
+Assign this issue to yourself and follow the below list.
+For any review action, review the file(s) in the issue/PR listed above. 
+Put comments in the above issue or PR if they concern non-changed files/text.
```

Comment:
I've just done the review in https://github.com/camaraproject/ReleaseManagement/issues/57 and thought that it is good to add a short review summary in addition to ticking the points of: 

```suggestion
Put comments in the above issue or PR if they concern non-changed files/text.
Put a short summary of the main review result here into the review issue.
```

Reply:
Maybe a bit double ? the comments are in the PR and in the issue, but ok if short.